testsuite/utils.py

```diff
@@ -62,7 +68,13 @@ def blame(request: 'FixtureRequest', name: str, tail: int = 3) -> str:
     if "." in context:
         context = context.split(".")[0]
 
-    return randomize(f"{name[:8]}-{_whoami()[:8]}-{context[:9]}", tail=tail)
+    # When Sphinx search works with strings generated by blame(), parts separated by '-' symbol
```

Comment:
My understanding is that described behavior is unexpected bug, we should not mitigate it thus, maybe we could rather mark affected test as xfail with reference to relevant bug.

On the other hand change above in `whoami` is nice one worth to merge, my suggestion is to rename commit remove changes in blame and repush it just with changes in `whoami`

Reply:
Made new PR: https://github.com/3scale-qe/3scale-tests/pull/300